lists2safebrowsing.py

```diff
@@ -49,7 +51,6 @@
     "tracking-protection-full",
     "staging-tracking-protection-full"
 )
-
 DNT_SECTIONS = (
```

Comment:
I have no idea what this section does. Did you find it's needed for the script to generate the list properly?

Reply:
Nevermind, I just figured out it's `DNT_SECTIONS` as opposed to `PRE_DNT_SECTIONS` (legacy lists).